docs/guides/sdk/getting-started.md

```diff
@@ -252,6 +252,71 @@ All SDK composables can also be conveniently imported from the root package `@di
 
 :::
 
+## Global APIs
+
+To keep the SDK dependency-free, it does rely on the APIs mentioned below, which originally came from the browser
+ecosystem and may not be available in all environments.
+
+#### The `fetch` API
+
+There are various reasons why you would want to overwrite or set `fetch` if it's not available. For example, Node.js
+version 16 and below do not yet ship `fetch`.
+
+- [`node-fetch`](https://github.com/node-fetch/node-fetch)
+- [`ofetch`](https://github.com/unjs/ofetch)
+- [`whatwg-fetch`](https://github.com/JakeChampion/fetch)
+
+#### The `URL` API
+
+This API is shipped in almost every runtime, but for example `react-native` does require a polyfill for the SDK to work.
+
+- [`url-polyfill`](https://github.com/lifaon74/url-polyfill)
+- [`react-native-url-polyfill`](https://www.npmjs.com/package/react-native-url-polyfill)
+
+#### The `WebSocket` API
+
+This API is optional if you're not making use of the `realtime()` features in the SDK. Backend JavaScript environments
+often do not ship with an implementation of WebSockets.
+
+- [`ws`](https://github.com/websockets/ws)
+- [`isomorphic-ws`](https://github.com/heineiuo/isomorphic-ws)
+
+### Polyfilling
+
+These can be overwritten in two ways:
+
+1. Via options parameter of the `createDirectus` function.
```

Comment:
Oh, should we mention that this takes precedence?

Reply:
Sure that makes sense